memory_analyzer/templates/analysis.py.template

```diff
@@ -82,6 +82,7 @@ try:
       pickle.dump(summ, fifo)
 
 except Exception as e:
+    print("Got exception", e)
```

Comment:
Does this show up? I thought it might get swallowed

Reply:
It shows up on whatever sys.stdout is of the target process in my testing.

Reply:
```
[tim@rygel ~]$ python -c 'import os; print(os.getpid()); import time; time.sleep(100)'
252143
Got exception No module named 'pympler'
```